workspace-variables/workspace.tfvars.example

```diff
@@ -61,3 +61,4 @@ google_maps_api_key =
 google_analytics =
 rollbar_access_token =
 domain = # "tvs.dxw.net"
+pp_transactions_by_channel_token =
```

Comment:
Shall we add `pp_user_satisfaction_token` to the example tfvars as well?

Reply:
yes, that sounds like a good idea 👍